tools/install-flatbuf.sh

```diff
@@ -7,20 +7,34 @@ retry() {
   $cmd || (sleep 2 && $cmd) || (sleep 10 && $cmd)
 }
 
-fetch_flatbuf() {
+fetch_flatbuf_unix() {
     rm -rf flatbuffers
     mkdir flatbuffers
     curl -L https://github.com/google/flatbuffers/archive/v1.11.0.tar.gz | tar -zx -C flatbuffers --strip-components 1
 }
 
+fetch_flatbuf_windows() {
+    curl -L https://github.com/google/flatbuffers/releases/download/v1.11.0/flatc_windows_exe.zip > fb.zip && unzip fb.zip
```

Comment:
windows comes with curl?

Reply:
Travis runs CI script in an environment that does. This is probably not going to work on all flavors of windows.